File: ArchiRedux/ArchiRedux/Framework/Dto/ProductDto.cs
using System;

namespace ArchiRedux.Framework.Dto
{
    public class ProductDto
    {
        public String Id { get; set; }
        public String Name { get; set; }
        public Int32 Price { get; set; }
        
        public Boolean IsRemoved { get; set; }

    }
}

File: ArchiRedux/ArchiRedux/ClientApp/src/model/productDto.ts
export class ProductDto {

    id: string;
    
    name: string;
    
    price: number;
    
    isRemoved: boolean;

    constructor(id: string, name: string, price: number, isRemoved: boolean) {
        this.id = id;
        this.name = name;
        this.price = price;
        this.isRemoved = isRemoved;
    }
}